libs/application/templates/children-residence-change/src/lib/messages/contract.ts

```diff
@@ -53,6 +40,18 @@ export const contract = {
       description: 'Label for new residence',
     },
   }),
+  childBenefit: defineMessages({
+    label: {
+      id: 'crc.application:section.contract.overview.childBenefit.label',
+      defaultMessage: 'Meðlag',
+      description: 'Label for child benefit',
+    },
+    text: {
+      id: 'crc.application:section.contract.overview.childBenefit.text',
+      defaultMessage: '{otherParent} fær einfalt meðlag mánaðarlega.',
```

Comment:
Does this line make sense if the parent has changing residence for multiple children?

Reply:
Yes it does because all the children will change to the same residence and therefore the child benefits will change for both children. After a team discussion we have decided to iterate on this later but this should not be a blocker for now.